cpu/cc26x2_cc13x2/radio/rf_commands.c

```diff
@@ -0,0 +1,188 @@
+/*
+ * Copyright (C) 2020 Locha Inc
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @ingroup         cpu_cc26x2_cc13x2
+ * @{
+ *
+ * @file
+ * @brief           CC13x2 IEEE 802.15.4 netdev driver
+ *
+ * @author          Jean Pierre Dudey <jeandudey@hotmail.com>
+ * @}
+ */
+
+#include "cc26x2_cc13x2_rf.h"
+#include "cc26xx_cc13xx_rfc_mailbox.h"
+
+/* Overrides for CMD_PROP_RADIO_DIV_SETUP */
+static uint32_t rf_prop_overrides[] =
+{
+    /* override_prop_common.xml
+     * DC/DC regulator: In Tx, use DCDCCTL5[3:0]=0x7 (DITHER_EN=0 and IPEAK=7).
+     */
+    (uint32_t)0x00F788D3,
+    /* override_tc146.xml
+     * Tx: Configure PA ramp time, PACTL2.RC=0x3 (in ADI0, set PACTL2[4:3]=0x3)
+     */
+    RFC_ADI_2HALFREG_OVERRIDE(0, 16, 0x8, 0x8, 17, 0x1, 0x1),
+    /* Tx: Configure PA ramping, set wait time before turning off (0x1A ticks
+     of 16/24 us = 17.3 us).
+     */
+    RFC_HW_REG_OVERRIDE(0x6028, 0x001A),
+    /* Rx: Set AGC reference level to 0x16 (default: 0x2E) */
+    RFC_HW_REG_OVERRIDE(0x609C, 0x0016),
+    /* Rx: Set RSSI offset to adjust reported RSSI by -1 dB (default: -2),
+     * trimmed for external bias and differential configuration */
+    (uint32_t)0x000188A3,
+    /* Rx: Set anti-aliasing filter bandwidth to 0x8 (in ADI0, set
+     * IFAMPCTL3[7:4]=0x8) */
+    RFC_ADI_HALFREG_OVERRIDE(0, 61, 0xF, 0x8),
+    /* TX power override
+     * Tx: Set PA trim to max to maximize its output power (in ADI0,
+     * set PACTL0=0xF8) */
+    RFC_ADI_REG_OVERRIDE(0, 12, 0xF8),
```

Comment:
I haven't moved them to proper defines because it's automatically generated by a TI tool, I don't know if it makes sense to manually change them. Sorry for not clarifying it with a comment on the code, should I move the magic values to defines?

Reply:
Ah then better add a comment that this code was auto-generated.